app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="editor-root">
+  <div class="editor-root relative">
```

Comment:
Is there any particular reason to make this relative? Is it to align the canned response selector?

Reply:
This is a mistake, editor root class already uses `relative`. Removing this change